api/v1/testkube.yaml

```diff
@@ -649,6 +718,137 @@ paths:
 
 components:
   schemas:
+
+    ObjectRef:
+      type: object
+      properties:
+        namespace:
+          type: string
+        name:
+          type: string
+
+
+    Test:
+      type: object
+      properties:
+        name: 
+          type: string
+        description: 
+          type: string
+        before:
+          type: array
+          items:
+            $ref: "#/components/schemas/TestStep"
+          description: Run this step before whole suite
+        steps:
+          type: array
+          items:
+            $ref: "#/components/schemas/TestStep"
+          description: Steps to run
+        after:
+          type: array
+          items:
+            $ref: "#/components/schemas/TestStep"
+          description: Run this step after whole suite
+        repeats:
+          type: integer
+          default: 1
+
+    TestStep: 
+      type: object 
+      required:
+        - name
+      properties:
+        name: 
+          type: string
+          description: step name (in case of script it'll be script name)
+
+    TestStepExecuteScript: 
+      allOf:     # Combines the BasicErrorModel and the inline model
+        - $ref: '#/components/schemas/TestStep'
+        - type: object
+          required:
+            - namepace 
+            - name
+          properties:
+            namespace:
+              type: string
+            name:
+              type: string
+            stopTestOnFailure:
```

Comment:
should this be in TestStep object?

Reply:
I assume delay don't have failures :) what can go wrong with delay :D ??

Reply:
sure - just trying to avoid step-specific processing as most other step types will probably be able to fail - the Delay step will probably always succeed (unless the test is aborted while delaying?)